WordPress/Classes/ViewRelated/Media/WPStyleGuide+Loader.swift

```diff
@@ -0,0 +1,25 @@
+
+import UIKit
+
+extension WPStyleGuide {
+    static func styleProgressViewWhite(_ progressView: MediaCellProgressView) {
```

Comment:
I tried this, you can check out the previous commit with this idea working. I felt like, as those styles depend on `WPStyleGuide`, it would make it more difficult to move this class to `WordPressUI`, so I decided to set the styles separately. But I'm open to test it again :)